07-lucid/01-model.adoc

```diff
@@ -353,6 +353,47 @@ const User = use('App/Models/User')
 const users = await User.reset()
 ----
 
+== Aggregate helpers
+Lucid aggregate helpers provide shortcut access to common aggregate queries without using the query builder interface. These same queries can be written long-form using the query builder interface. Short and long-form examples are provided for each helper method.
+
+There is no need to call `fetch` when using one of the following static methods. These methods end the query builder chaining and return a value.
+
+All helpers accept a column name to be used for aggregation. When possible Lucid will choose a default for the column name. Some methods, such as `sum()`, require a column name.
+
+==== rowsCount(columnName = '*')
+Return a count of records in a given result set.
+
+[source, js]
+----
+const User = use('App/Models/User')
+
+# Return count of all records in table
+await User.rowsCount()
+
+# Return count of selected records
+await User.query().rowsCount()
```

Comment:
I think we should add a `where` clause to this example so people will understand it better.